src/diskquota.h

```diff
@@ -217,6 +217,7 @@ struct DiskquotaDBEntry
 
 	bool inited; // this entry is inited, will set to true after the worker finish the frist run.
 	bool in_use; // this slot is in using. AKA dbid != 0
+	bool corrupted; // consider this entry as invalid to start the worker on
```

Comment:
Clang-format CI check over my changes at `diskquota.h` has failed. Should I fix it?

Reply:
Clang-format complains to upstream changes, not your changes:

```
-	bool inited; // this entry is inited, will set to true after the worker finish the frist run.
-	bool in_use; // this slot is in using. AKA dbid != 0
+	bool inited;    // this entry is inited, will set to true after the worker finish the frist run.
+	bool in_use;    // this slot is in using. AKA dbid != 0
 	bool corrupted; // consider this entry as invalid to start the worker on
```

Reply:
Well, here, In order to pass all the checks, I eventually [edited](https://github.com/arenadata/diskquota/commit/3b06e37e4882b4a3d95448df1c944ded79ff9c07#diff-2f4057fef3b75c8591966cbaf2e4f6537a89038ae371d22d2d0abd65db495831) all those lines

Reply:
I filtered out unintentionally these changes when reapply your patch in f280a884c6056ad0d89a34c80b91485cf360f3ab. Let's leave it for upstream. We shouldn't reformat each patch.